testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/hadoop/hive_with_run_view_as_invoker.properties

```diff
@@ -0,0 +1,9 @@
+connector.name=hive
+hive.metastore.uri=thrift://hadoop-master:9083
+hive.config.resources=/docker/presto-product-tests/conf/presto/etc/hive-default-fs-site.xml
+hive.metastore-cache-ttl=0s
+hive.fs.cache.max-size=10
+hive.max-partitions-per-scan=100
+hive.hive-views.enabled=true
+hive.hive-views.run-as-invoker=true
+hive.security=sql-standard
```

Comment:
redundant?

Reply:
Nope, I am actually using this one when doing `GRANT`